plugins/easyimage/styles/easyimage.css

```diff
@@ -42,3 +42,17 @@ For licensing, see LICENSE.md or http://ckeditor.com/license
 	border: 1px solid #48a3f5;
 	outline: none;
 }
+
+/* Loaders */
+
+.cke_loader {
+	position: absolute;
+	left: 0px;
+	right: 0px;
+}
+
+.cke_loader .cke_bar {
+	height: 10px;
```

Comment:
I like CKE5 loader, which is much thinner. Maybe we could also make it thin?

Reply:
I had a problem with it that it's _barely visible_, I didn't see it at all at first. Let's extract beautifying / styling into a separate user story where Ryszard will help us.